### ISSUE TITLE
Add a license to the repo

### ISSUE THREAD

**Issue by Kazade** [pspdev/psplibraries#75]
Although the individual libraries have their own licenses, the scripts, patches, etc. in this repo don't have a license (as far as I can tell).

Specifically I'm looking at distributing the PSP.cmake file, and unless it has a license I can't legally do that :(

**Comment by carstene1ns** [comment]
You are right about that. We were in the process of changing the repository content (how the libraries are built), after that we should come up with one.
About the individual licenses to the cmake stuff, it is best to ask their authors (:arrow_right: @dbeef) in the meanwhile.

**Comment by sharkwouter** [comment]
Actually, the license for PSP.cmake was decided to be GPL in a chat with dbeef and wally. It was added to PSPBUILD was was made in an effort to change the way libraries are build: https://github.com/sharkwouter/psplibraries/blob/pspbuilds/psp-cmake/PSPBUILD
